api-report/container-runtime.api.md

```diff
@@ -70,8 +71,10 @@ export enum ContainerMessageType {
     Rejoin = "rejoin"
 }
 
+// Warning: (ae-forgotten-export) The symbol "IGarbageCollectionRuntime" needs to be exported by the entry point index.d.ts
```

Comment:
Does this comment need to be addressed?

Reply:
I believe so! Wonder if this should be a compile error instead. Easy to miss!